cameraview/src/main/java/com/otaliastudios/cameraview/engine/Camera1Engine.java

```diff
@@ -696,6 +697,11 @@ private boolean applyPlaySounds(boolean oldPlaySound) {
         return false;
     }
 
+    @Override
+    public void setPreviewFrameRateExact(boolean previewFrameRateExact) {
+        mPreviewFrameRateExact = previewFrameRateExact;
+    }
+
```

Comment:
I would prefer if this was done in `CameraBaseEngine` (abstract setter and getter in `CameraEngine`, then final setter and getter in `CameraBaseEngine`, then make `mPreviewFrameRateExact` private there. The two engines can call getPreviewFrameRateExact() to check the value)